lightning-background-processor/src/lib.rs

```diff
@@ -788,7 +791,10 @@ mod tests {
 
 			if key == "network_graph" {
 				if let Some(sender) = &self.graph_persistence_notifier {
-					sender.send(()).unwrap();
+					match sender.send(()) {
```

Comment:
This is in only in tests, just wanted to check that the intention was to fix test related things as the commit name didn't imply this.

Reply:
That's true!

I cherry-picked this from `lightningdevkit/rust-lightning` and I guess these lines were included too. That's why `tnull` is a co-author here. I would just leave them in, but the rest of the patch is what is actually important.

Btw, these changes will be there when we update to 0.0.115, as well as some of the other changes in this PR.